server/execution/entities/execution.py

```diff
@@ -49,3 +51,17 @@ def to_json(self, shallow: bool = False):
         only the object reference in form of a unique identifier is included.
         """
         return json.dumps(self.to_dict(shallow))
+
+    def add_new_player(self, role: int, location: int):
+        from utils.tans import uniques
+        from execution.run import register_player
+        from execution.services.entityloader import load_location
+        from execution.services.entityloader import load_role
+        tan = str(uniques(1)[0])
```

Comment:
Doesn't `unqiues` return strings already?

And do you mind, if I refactor this later and create an additional method in the Tan-class that returns a single unique TAN, as this is probably the most-likely use case?

Reply:
Nevermind, returns TANs obviously 😅

Reply:
Nope, uniques returns a list of Tans (the object). And no, I don't mind as I do have missed that functionality myself :)